test/e2e/README.md

```diff
@@ -192,6 +192,49 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been pushed, the following steps should be done for running the e2e tests.
```

Comment:
```suggestion
If you already have a K8s cluster on which Antrea has been deployed, the following steps should be done for running the e2e tests.
```

Why did you use "push"? I guess you are referring to the CI process but since this section is about manually testing so I suggest "deploy".